test/Lint.lean

```diff
@@ -86,3 +86,32 @@ note: this linter can be disabled with `set_option linter.cdot false`
 #guard_msgs in
 set_option linter.cdot true in
 example : Add Nat where add := (. + ·)
+
+set_option linter.longLine false
+/--
+warning: This line exceeds the 100 character limit, please shorten it!
+note: this linter can be disabled with `set_option linter.longLine false`
+-/
+#guard_msgs in
+set_option linter.longLine true in
+/-!                                                                                                -/
+
+#guard_msgs in
+-- Lines with more than 100 characters containing URLs are allowed.
+set_option linter.longLine true in
+/-!  http                                                                                          -/
+
+set_option linter.longLine true
+-- check that the *argument* of `#guard_msgs` is *not* exempt from the linter
+/--
+warning: This line exceeds the 100 character limit, please shorten it!
+note: this linter can be disabled with `set_option linter.longLine false`
+-/
+#guard_msgs in                                                                            #guard true
+
+-- check that the doc-string of `#guard_msgs` is exempt from the linter
```

Comment:
(perhaps with additional line wrapping)
```suggestion
-- However, the *doc-string* of `#guard_msgs` is exempt from the linter: these are automatically generated, hence linting them is not helpful.

Reply:
I added this manually, since otherwise the long line linter complains. :smile: